csi/node_server.go

```diff
@@ -136,16 +139,16 @@ func (ns *NodeServer) nodePublishMountVolume(volumeName, devicePath, targetPath,
 
 func (ns *NodeServer) nodePublishBlockVolume(volumeName, devicePath, targetPath string, mounter *mount.SafeFormatAndMount) (*csi.NodePublishVolumeResponse, error) {
 	targetDir := filepath.Dir(targetPath)
-	exists, err := mounter.ExistsPath(targetDir)
+	exists, err := hostUtil.PathExists(targetDir)
 	if err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 	if !exists {
-		if err := mounter.MakeDir(targetDir); err != nil {
+		if err := makeDir(targetDir); err != nil {
```

Comment:
Was this done in the Pod's namespace or host namespace?

Reply:
From what I am understanding, it is the same whether we execute this command in Pod's namespace or host namespace because we Bidirectionally mount `/var/lib/kubelet/pods` on the Pod to `/var/lib/kubelet/pods` on the host. How do you think?

On the other hand, we currently are using [mounter.MakeDir(targetDir)](https://github.com/longhorn/longhorn-manager/blob/9f3059094d3359dfdd00b3727989aaf9fd758732/vendor/k8s.io/kubernetes/pkg/util/mount/mount_linux.go#L389)  which does the same thing as the new one `makeDir ()`